indico/modules/events/papers/forms.py

```diff
@@ -16,11 +16,27 @@
 
 from __future__ import unicode_literals
 
+from datetime import time
+from wtforms.validators import Optional
+
 from indico.util.i18n import _
 from indico.web.forms.base import IndicoForm
+from indico.web.forms.fields import IndicoDateTimeField
 from indico.web.forms.fields import PrincipalListField
+from indico.web.forms.validators import LinkedDateTime
 
 
 class PaperManagersForm(IndicoForm):
     managers = PrincipalListField(_('Paper managers'), groups=True,
                                   description=_('List of users allowed to manage the call for papers'))
+
+
+class PapersScheduleForm(IndicoForm):
+    start_dt = IndicoDateTimeField(_("Start"), [Optional()], default_time=time(0, 0),
+                                   description=_("The moment users can start submitting papers"))
+    end_dt = IndicoDateTimeField(_("End"), [Optional(), LinkedDateTime('start_dt')], default_time=time(23, 59),
+                                 description=_("The moment the submission process will end"))
```

Comment:
I'd suggest *The moment the submission process ~will~ **ends*** cause using the future tense will look strange if this deadline has passed and the description text still says *will*.